app/Http/Controllers/PostsController.php

```diff
@@ -71,6 +81,9 @@ public function show($id)
     public function edit($id)
     {
         $post = Post::find($id);
+        if(auth()->user()->id !== $post->user_id){
```

Comment:
I don't see include("message.blade.php") here, so it is not currently used.

Reply:
mmmh... I'm not sure if I need to do this part now... if the message.blade is needed to be, where should it be?
Write Story: add the message where it suppose to show. Error, Success, Delete so on.